test/benchmark_http_client_test.cc

```diff
@@ -220,33 +220,6 @@ TEST_F(BenchmarkClientHttpTest, StatusTrackingInOnComplete) {
   client_.reset();
 }
 
-TEST_F(BenchmarkClientHttpTest, ConnectionPrefetching) {
```

Comment:
Do we have any new behaviors that we should test here? Say the difference between having the setPrefetchConnections enabled and disabled?

Reply:
Sorry, should have probably mentioned this in the PR description, but the test at https://github.com/envoyproxy/nighthawk/blob/master/test/integration/test_integration_basics.py#L246 is also in place.

That python-based integration testing is often convenient for end-to-end tests, but was not around initially. With the changes made here, this C++ test needed maintenance, but I figured it didn't really add any value on top of the python-based test. So I dropped it instead.

Reply:
Thank you @oschaaf.